rootfs/api/models/build.py

```diff
@@ -43,10 +43,11 @@ def create(self, user, *args, **kwargs):
         try:
             self.app.deploy(new_release)
             return new_release
-        except Exception:
+        except Exception as e:
             if 'new_release' in locals():
                 new_release.delete()
-            raise
+
+            raise DeisException(str(e)) from e
```

Comment:
would this mask any important errors? I feel like we'd be losing the exception class that was raised in a traceback if we're converting it straight to a DeisException.